src/NzbDrone.Core/Datastore/Migration/113_music_blacklist.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(113)]
+    public class music_blacklist : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Blacklist")
+                  .AddColumn("ArtistId").AsInt32().WithDefaultValue(0)
```

Comment:
The snippet below is from the housekeeper, that's looking for artists that have already been deleted (Artist Id is null in the artist table), but for some reason the blacklist item didn't get removed. It then "cleans" the db of that record.